apps/gallery/index.html

```diff
@@ -209,11 +210,11 @@ <h1 id="edit-title" data-l10n-id="edit"></h1>
         </div>
 
         <div id="edit-crop-options" class="edit-options-bar hidden" role="radiogroup" data-l10n-id="crop-options">
-          <a id="edit-crop-none" class="button" data-icon="undo" role="button" data-l10n-id="crop-revert"></a>
           <a id="edit-crop-aspect-free" class="selected radio button" role="radio" aria-checked="true" data-l10n-id="crop-aspect-free"></a>
           <a id="edit-crop-aspect-portrait" class="radio button" role="radio" data-l10n-id="crop-aspect-portrait"></a>
           <a id="edit-crop-aspect-landscape" class="radio button" role="radio" data-l10n-id="crop-aspect-landscape"></a>
           <a id="edit-crop-aspect-square" class="radio button" role="radio" data-l10n-id="crop-aspect-square"></a>
+          <a id="edit-crop-aspect-more" class="radio button" role="radio" data-icon="more" role="button" data-l10n-id="crop-aspect-more"></a>
```

Comment:
This <a> element for the more button has two role="" attributes, which doesn't seem like it can be right.